build.gradle

```diff
@@ -41,6 +41,12 @@ allprojects {
 		testCompile 'org.hamcrest:hamcrest-library:2.1'
 		testCompile 'org.mockito:mockito-core:2.28.2'
 
+		testCompile group: 'junit', name: 'junit', version: '4.8.2'
+		testCompile group: 'com.diffblue', name: 'deeptestutils', version: '1.9.0'
+		testCompile group: 'powermock', name: 'powermock', version: '1.7.4'
+		testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: '1.7.4'
+		testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: '1.7.4'
```

Comment:
Powermock is not used

Reply:
This is still not resolved, you need to remove powermock from the dependencies as it's not used in the tests

Reply:
Yeah forgot to commit updated build.gradle, should be removed now

Reply:
Unless my eyes are deceiving me, it's still there.